packages/react-native-bpk-component-calendar/README.md

```diff
@@ -36,7 +37,7 @@ const minDate = new Date(Date.UTC(2019, 0, 2));
 To format the dates for display use
 
 ```javascript
-const locale = 'en-GB';
+const locale = 'en-gb';
```

Comment:
[ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) codes should be capitalized by convention I believe